expressEngine/orderItems/service/orderItems.js

```diff
@@ -0,0 +1,30 @@
+// import service libraries
+import { BaseService, DbService } from '../../lib/service/index';
```

Comment:
It is exported as `DBService`, so please replace `DbService` with `DBService`.

Reply:
Please remove the index from import statement.

Reply:
There is no need to import from index